packages/js/product-editor/src/components/variations-table/downloads-menu-item/downloads-menu-item.tsx

```diff
@@ -199,6 +201,10 @@ export function DownloadsMenuItem( {
 					>
 						{ __( 'Set download expiry', 'woocommerce' ) }
 					</MenuItem>
+					<VariationQuickUpdateMenuItem.Slot
+						group={ 'downloads' }
+						supportsMultipleSelection={ supportsMultipleSelection }
+					/>
```

Comment:
We need to pass through the `selection` and `onChange` so that fills can access that in their `onClick` handler and actually do something!

We should also pass  the`onClose` callbacks, as that will be needed too for extensions to close the menu when they have done their action!